docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/feature-howtos.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Feature HowTos
+description: This section lists documents inside the Feature HowTos section.
+last_updated: Apr 26, 2023
+template: concept-topic-template
+---
+
+The *Feature HowTos* section includes the following documents:
```

Comment:
Can you add a few words on what this section is about?

Reply:
```suggestion
The "Feature HowTos" section provides step-by-step instructions about adding support for number formatting in both the Storefront and Back Office user interfaces. To integrate and implement number formatting, follow the detailed instructions in this section:
```